server/engine/Card.js

```diff
@@ -0,0 +1,17 @@
+class Card {
+  constructor(word, role) {
+    this.word = word;
+    this.role = role;
+    this.selected = false;
+  }
+
+  select() {
+    if (this.selected) {
+        return false;
```

Comment:
I'm just curious as to why select() is returning false (seems like a setter method) in this first conditional when a card is selected. I think a player should be able to deselect his choice of card.

Reply:
To my understanding, if the player select the card, it will instantly show the role of card. Maybe it should be front end's responsibility to disable the card from being clickable. Correct me if I'm wrong.

Reply:
Oh, I see. I think you're right then. I just find it odd that it's returning false here and nothing else gets returned outside of this block in the function. 

Also, I think we have to consider the use-case where the team has to vote on a card to be selected on the board before it actually gets selected (as shown on field agent's UI mockup).

Reply:
You are right. Return value of this function is not being used now. I'm not sure how the voting thing work right now, it's not outlined in the game engine v1. Maybe we will know more after today's sprint.